src/renderer/components/commands-action-button.tsx

```diff
@@ -180,7 +193,7 @@ export const GistActionButton = observer(
     /**
      * Update an existing GitHub gist.
      */
-    public async handleUpdate() {
+    public async handleUpdate(isPublishingGistAsRevision = false) {
```

Comment:
nit: we could rename this parameter as `silent` to better understand what it's supposed to do as opposed to why we're doing it.